package.json

```diff
@@ -29,9 +29,9 @@
     "@fontsource/open-sauce-sans": "^5.0.7",
     "@headlessui/react": "1.7.19",
     "@heroicons/react": "^2.1.3",
-    "@indexcoop/analytics-sdk": "0.17.0",
-    "@indexcoop/flash-mint-sdk": "3.2.1",
-    "@indexcoop/tokenlists": "1.46.0",
+    "@indexcoop/analytics-sdk": "0.19.0",
```

Comment:
need this release first before merging? https://github.com/IndexCoop/analytics-sdk/pull/39#pullrequestreview-2138553327

Reply:
added